evalml/preprocessing/data_splitters/balanced_classification_sampler.py

```diff
@@ -68,6 +71,22 @@ def _find_ideal_samples(self, y):
         drop_values = {k: max(0, counts[k] - goal_value) for k in undersample_classes}
         return {k: v for k, v in drop_values.items() if v > 0}
 
+    def _sampling_dict_to_remove_dict(self, y):
+        """Turns the sampling dict input into a dict of samples to remove for each target, similar to the return of _find_ideal_samples.
+
+        Arguments:
+            y (pd.Series): Training data targets
+
+        Returns:
+            (dict): dictionary with undersample target class as key, and number of samples to remove as the value.
+                If we don't need to resample, returns empty dictionary.
+        """
+        y_dict = y.value_counts().to_dict()
+        new_dic = {}
+        for k, v in self.sampling_ratio_dict.items():
+            new_dic[k] = max(y_dict[k] - v, 0)
```

Comment:
@bchen1116 The doc said that we'd accept ratios as the values of the dictionary but it seems like we're still expecting counts. Is that coming in the part 2 pr?

Reply:
Got it. I think I understand the plan now!